src/main.py

```diff
@@ -1,7 +1,15 @@
+import os
+
 from follower import Follower
 from leader import Leader
 from tagrec import TagRecognition
 
+"""
+Returns whether or not a Bluetooth joystick is connected.
+"""
+def is_connected():
```

Comment:
is_connected to what? The internet? Connected like a smart-fridge? Does this function tell me if it's IoT? Try `is_controller_connected`